Wrapping/CMakeLists.txt

```diff
@@ -36,6 +36,39 @@ project(WrapITK
 
 enable_testing()
 
+###############################################################################
+# Set build configuration options
+###############################################################################
+if(CMAKE_CONFIGURATION_TYPES)
+  option(ALLOW_MULTICONFIG_WRAPPING "Allow (dangerous) multi-config wrapping, USE WITH CAUTION!" OFF)
+  mark_as_advanced(ALLOW_MULTICONFIG_WRAPPING)
+
+  if(ALLOW_MULTICONFIG_WRAPPING)
+    if(NOT CMAKE_BUILD_TYPE)
+      message(FATAL_ERROR "Wrapping requires explicit setting of CMAKE_BUILD_TYPE")
+    endif()
+    list(LENGTH CMAKE_CONFIGURATION_TYPES NUM_CMAKE_CONFIGURATION_TYPES)
+    if(NOT NUM_CMAKE_CONFIGURATION_TYPES EQUAL 1)
+      message(WARNING "CMake multi-config generators are not supported with ITK_WRAPPING:BOOL=ON.
```

Comment:
I did not see this warning despite having CMAKE_BUILD_TYPE=Release and CMAKE_CONFIGURATION_TYPES=RelWithDebInfo.

Reply:
This was disabled by default.  Perhaps it should be enabled by default?

Reply:
If it leads to problems, it definitely should. I had no need to fiddle with CMAKE_BUILD_TYPE as it did not affect anything in multi-config generators. So it always had the default value.

Reply:
This was more important earlier, then I found a way to make the process more clean.

I was on the fence about completely removing this.

If the building work, I propose that we just remove this logic.